mission.md

```diff
@@ -0,0 +1,332 @@
+[![woman-on-a-mission-intro-image](https://user-images.githubusercontent.com/194400/68089526-a0871180-fe61-11e9-82b2-d4f63872428e.jpg
+  "credit: https://unsplash.com/photos/pPFZibBBLGc")](https://unsplash.com/photos/pPFZibBBLGc)
+<!-- if you can find a better intro image, please share! -->
+
+
+# Mission
+
+People with a clearly communicated `Mission`
+are more likely to achieve their aim. <br />
+
+There is a clear correlation
+between having a _concise_ mission
+and the _successful outcome_
+of the people
+focussed on achieving it.
+In many cases feats
+previously considered "_impossible_" are achieved
+in _unbelievably_ short timelines.
+
+
+## Context
+
+We have always had a
+["_Why_?"](https://github.com/dwyl/start-here#why)
+goal (_or several goals depending on who you ask_) for @dwyl.
+
+_Long_ before we started @dwyl we had been working for charities
+and non-profit organisations on a variety of missions
+including poverty reduction, cancer research, homelessness,
```

Comment:
I'll submit a PR to add environmental concerns to this section 🙌

Reply:
Yes, please do. 👍